lib/BasicAnalyses/RemoveNewPCCalls.cpp

```diff
@@ -0,0 +1,30 @@
+/// \file RemoveNewPCCalls.cpp
+/// \brief Remove calls to newpc in a function.
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/ADT/SmallVector.h"
+
+#include "revng/BasicAnalyses/RemoveNewPCCalls.h"
+#include "revng/Support/IRHelpers.h"
+
+llvm::PreservedAnalyses
+RemoveNewPCCallsPass::run(llvm::Function &F,
+                          llvm::FunctionAnalysisManager &FAM) {
```

Comment:
I think we already have this pass in revng-c.
Can you review the differences and import the one from revng-c here?

Reply:
Already done when imported from `revng-c`.

Reply:
Can you make a draft MR to revng-c to drop its version in favor of this one?
You can undraft it once this PR goes in.